util/opentelemetry-util-genai-dev/examples/traceloop_span_transformation/traceloop_rules_example.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+
+import json
+import os
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import (
+    SimpleSpanProcessor,
+    ConsoleSpanExporter,
+)
+
+from opentelemetry.util.genai.processors.traceloop_span_processor import (
+    TraceloopSpanProcessor,
+)
```

Comment:
This doesn't compile for me. Wondering if I'm doing something wrong. Tried to fix it but it caused problems elsewhere.
```
from opentelemetry.util.genai.processors.traceloop_span_processor import (
ModuleNotFoundError: No module named 'opentelemetry.util.genai.processors'
```

Reply:
Maybe a Pycharm issue because this file runs at the CLI.